### DISCUSSION TITLE
Welcome to Scalene Discussions!

### DISCUSSION THREAD

**Discussion by emeryberger** [plasma-umass/scalene#96] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sheerluck** [comment]
Hi, I just wonder is it possible to package scalene for Gentoo Linux? Thanks.

**Reply by emeryberger** [reply]
Hi @sheerluck - is `pip` not an option?
